### ISSUE TITLE
MXNET on PASCAL GPUS

### ISSUE THREAD

**Issue by amithr1** [dmlc/mxnet#4526]
Hi All,

Does MXNET work on PASCAL nvidia GPUs? (GP100GL)?
When I try a CIFAR test, I get the following segfault. Is there any flags I can enable to debug this? 
I already tried DEBUG=1

-bash-4.2$ python example/image-classification/train_cifar10.py --kv-store device --gpus 0 
INFO:root:start with arguments Namespace(batch_size=128, benchmark=0, data_nthreads=4, data_train='data/cifar10_train.rec', data_val='data/cifar10_val.rec', disp_batches=20, gpus='0', image_shape='3,28,28', kv_store='device', load_epoch=None, lr=0.1, lr_factor=0.1, lr_step_epochs='200,250', max_random_aspect_ratio=0, max_random_h=36, max_random_l=50, max_random_rotate_angle=0, max_random_s=50, max_random_scale=1, max_random_shear_ratio=0, min_random_scale=1, model_prefix=None, mom=0.9, network='resnet', num_classes=10, num_epochs=300, num_examples=50000, num_layers=110, optimizer='sgd', pad_size=4, random_crop=1, random_mirror=1, rgb_mean='123.68,116.779,103.939', test_io=0, top_k=0, wd=0.0001)
[10:29:41] src/io/iter_image_recordio.cc:221: ImageRecordIOParser: data/cifar10_train.rec, use 4 threads for decoding..
[10:29:42] src/io/iter_image_recordio.cc:221: ImageRecordIOParser: data/cifar10_val.rec, use 4 threads for decoding..
INFO:root:in get lr scheduler, epoch_size: 390
INFO:root:Start training with [gpu(0)]
Segmentation fault

**Comment by mli** [comment]
p100 requires cuda8.0, (also the corresponding cudnn with cuda8.0). besides, we need to add the flag `-gencode arch=compute_60,code=compute_60` for `CUDA_ARCH` defined in config.mk
